src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="container__calendar calendar calendar--month-lenth-30">
+        <div class="calendar__day calendar__day-start--tue"></div>
```

Comment:
start day modifiler should be on calendar block, not a day. Your markup should looks like this one
```
<div class="calendar calendar--start-day-tue calendar--month-length-30">
  <div class="calendar__day"></div>
  <div class="calendar__day"></div>
</div>
```